### PR TITLE
Add Interactive list to the project

### PR DESCRIPTION
**Pull request by Paskab012**
# To Do list: interactive list

### Hello Reviewer🤠,
During this project, I have sure to follow all the requirements by : 

- Add a new JavaScript file and import it as a module:
it will contain methods related to the status updates (completed: true/false).
- Add event listener to the checkbox (change).
- Update items object's value for completed key upon user actions.
- Implement a function for the "Clear all completed" button (use filter() method).
- Store the updated array of items in local storage, so the user gets the correct list values after the page reloads.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /* eslint-disable import/extensions */
```
Comment (medaminedev66) [inline src/index.js]: - [ ] Kindly don't disable linters in your code. A linter is a tool that analyzes your source code to flag programming errors, bugs, stylistic errors, and suspicious constructs. Check this link for more information [Wikipedia](https://en.wikipedia.org/wiki/Lint_(software)).

**Review by medaminedev66 — CHANGES_REQUESTED** [review]
Hi @KABIKA681 , :wave:

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

_See the inline comments below for more required changes_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js [hunk 2]
```diff
@@ -1,6 +1,5 @@
-/* eslint-disable import/extensions */
```
Comment (mateo951) [inline src/index.js:1]: Addressed ✔

**Review by mateo951 — APPROVED** [review]
### Hi @KABIKA681, 

Well done implementing all the changes requested by your previous reviewer 🙌
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔥 
### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**